Add EXT_meshopt_compression and KHR_mesh_quantization support

kikijiki opened the pull request.

kikijiki committed:
Add EXT_meshopt_compression and KHR_mesh_quantization support

kikijiki committed:
fix texcoords

jaynewey commented:
Just to note: `KHR_mesh_quantization` doesn't seem to work when `KHR_texture_transform` is also applied

kikijiki committed:
Fix u8/u16 normalization, add tests, fmt

kikijiki commented:
@jaynewey Thanks for catching that. The library wasn't tracking the normalized flag for U8/U16 texture coordinates, it always normalized them. I fixed that and added a couple of small tests.
